packages/wrangler/src/bundle.ts

```diff
@@ -395,13 +396,23 @@ export async function bundleWorker(
 		_path.includes(".map")
 	)[0];
 
+	const resolvedEntryPointPath = path.resolve(
+		entry.directory,
+		entryPointOutputs[0][0]
+	);
+
+	// copy all referenced modules into the output bundle directory
+	for (const module of moduleCollector.modules) {
+		await writeFile(
```

Comment:
Since we are awaiting each of these and running them serially (not in parallel) there is no performance benefit really of using the async version of `writeFile()`.

```suggestion
		fs.writeFileSync(
```